maintainers/scripts/check-maintainer-github-handles.sh

```diff
@@ -4,20 +4,19 @@
 # Example how to work with the `lib.maintainers` attrset.
 # Can be used to check whether all user handles are still valid.
 
-set -e
-
-# nixpkgs='<nixpkgs>'
-# if [ -n "$1" ]; then
+set -o errexit -o noclobber -o nounset -o pipefail
```

Comment:
That is so explicit that most people will not know what this means. I guess:

errexit is -e and nounset is -u?

Reply:
> That is so explicit that most people will not know what this means.

That sentence doesn't make sense to me:

- How can something *more explicit* be harder to understand, unless you've only ever seen the abbreviation?
- Someone with no experience of either is going to find the longer option easier to read than the abbreviation, and might even get an inkling of what it actually means when seeing more than a single character.
- It is easy to search for the long version using a search engine or `man bash`. The single character abbreviation, not so much.

Reply:
> How can something more explicit be harder to understand, unless you've only ever seen the abbreviation?

That answers itself. If you only ever have seen the short variant you don't know it.

> Someone with no experience of either is going to find the longer option easier to read than the abbreviation, and might even get an inkling of what it actually means when seeing more than a single character.

The bash options are not self describing and someone having no experience will not understand either.

> It is easy to search for the long version using a search engine

Since I am not the only person using the short options you will find a lot of results when searching https://www.google.com/search?q=set+-euo+pipefail

> or man bash. The single character abbreviation, not so much.

Thats true. The bash man page is just to long.